gui/default/syncthing/folder/editIgnoresModalView.html

```diff
@@ -14,12 +14,13 @@
     </dl>
   </div>
   <div class="modal-footer">
-    <div class="pull-left"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
-    <button type="button" class="btn btn-primary btn-sm" ng-click="saveIgnores()" data-dismiss="modal">
+    <div class="pull-left" ng-show="editingExisting"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
+    <div class="pull-left" ng-show="!editingExisting"><span translate>Creating ignore patterns, overwriting an existing file at</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code>.</div>
```

Comment:
Never mind, rather do it right than do it twice. Two things:

Currently the path is formatted as `<code>` which is not possible within a translated string due to https://github.com/syncthing/syncthing/blob/master/gui/default/syncthing/app.js#L27. I guess this can't/shouldn't be changed for security reasons. So just add the path unformatted?

And lets get rid of this historical accident then. As this already introduces a new string, a second one (well an additional changed one) shouldn't hurt much, right?